backend/src/packages/chats/chat.service.ts

```diff
@@ -73,10 +73,19 @@ class ChatService implements Service {
     return chat;
   }
 
-  public createMessage(
+  public async createMessage(
     payload: ChatMessageCreatePayload,
   ): Promise<ChatMessageGetAllItemResponseDto> {
-    return this.chatMessageService.create(payload);
+    const createdMessage = await this.chatMessageService.create(payload);
+
+    const { chatId, updatedAt } = createdMessage;
+
+    await this.chatRepository.update({
+      id: chatId,
+      updatedAt: updatedAt.toDateString(),
+    });
```

Comment:
Not sure whether it is needed

Reply:
@v0ldemar01 Because for now, chat doesn't update if there is a new message in it, and I am sure that it is incorrect. If I did this, I update its updatetAt, so I can send all chats in the DESC way and this is a requirement for this task - "Chats with the newest messages should be shown on top of the chat list"

Reply:
You can just sort using db query without setting new value to updatedAt

Reply:
@v0ldemar01 But how can I sort it, if updatetAt never updates?
Sort without it just send chats in order when they were created, not updated by a new message.

Reply:
Just make a join with messages and sort by messages

Reply:
Will try to reach it